qiskit/circuit/quantumcircuit.py

```diff
@@ -2265,17 +2265,98 @@ def reset(self, qubit: QubitSpecifier) -> InstructionSet:
         return self.append(Reset(), [qubit], [])
 
     def measure(self, qubit: QubitSpecifier, cbit: ClbitSpecifier) -> InstructionSet:
-        """Measure quantum bit into classical bit (tuples).
+        """Measure quantum bit (qubit) into classical bit (cbit).
```

Comment:
```suggestion
        """Measure one or more quantum bits (qubit) into classical bits (cbit).
```
(or "measure _a_ quantum bit into _a_ classical bit", depending on whether you want to stress the broadcasting or not)

Reply:
I went for the singular option in daaed65035ffba33e2b3627a539f0207388e8a70